protobuf/can_do_crypto.proto

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021 Contributors to the Parsec project.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+syntax = "proto3";
+
+package can_do_crypto;
+
+import "psa_key_attributes.proto";
+
+enum CheckType {
+  Use = 0;
+  Generate = 1;
+  Import = 2;
+  Derive = 3;
```

Comment:
Enum names should be all-caps, as per [the style guide](https://developers.google.com/protocol-buffers/docs/style#enums)